crates/subspace-node/src/lib.rs

```diff
@@ -65,6 +67,9 @@ pub enum Subcommand {
     /// Import blocks.
     ImportBlocks(sc_cli::ImportBlocksCmd),
 
+    /// Import blocks from network.
+    ImportBlocksFromNetwork(ImportBlocksFromNetworkCmd),
```

Comment:
How about `ImportBlocksFromDSN`? Should be more explicit and DSN is not uncommon for a blockchain storage related project.